styles/wrap-guide.less

```diff
@@ -7,6 +7,15 @@ atom-text-editor::shadow {
     z-index: 3;
     position: absolute;
     top: 0;
-    background-color: @syntax-wrap-guide-color;
+    background-color: transparent;
+
+    &:after {
```

Comment:
That's the only way to get it to be 5 pixels wide, yet have the dotted border located down the middle of the element. Tooltips don't trigger on margins, since it's technically not part of the element's visible region. So while this is not really needed for this PR, this construction makes it possible to easily merge this one with #37. I'm pretty convinced we're going to merge #37, so I wanted to save myself some work :smirk: